integrationTests/factory/constants.go

```diff
@@ -7,6 +7,7 @@ const (
 	PrefsPath             = "../../../cmd/node/config/prefs.toml"
 	ExternalPath          = "../../../cmd/node/config/external.toml"
 	P2pPath               = "../../../cmd/node/config/p2p.toml"
+	EpochPath             = "../../../cmd/node/config/enableEpochs.toml"
```

Comment:
You could define a constant for base node's configuration path:
`baseNodeConfigPath = "../../../cmd/node/config"`
Then refactor these paths similar to:

`PrefsPath = fmt.Sprintf("%s/%s", baneNodeConfigPath, "prefs.toml")`